--- XQilla-2.3.2/Makefile.am.orig	2016-01-29 20:21:46.349549168 +0100
+++ XQilla-2.3.2/Makefile.am	2016-01-29 20:52:35.932804878 +0100
@@ -17,7 +17,7 @@
 
 # Library version, see http://sourceware.org/autobook/autobook/autobook_91.html
 # -no-undefined for Cygwin (bug #1639669)
-libxqilla_la_LDFLAGS = -version-info 6:0:2 -no-undefined
+libxqilla_la_LDFLAGS = -version-info 6:2:0 -no-undefined
 
 libxqilla_la_SOURCES = \
 src/framework/ReferenceCounted.cpp \
